pkg/issuer/acme/dns/rfc2136/rfc2136.go

```diff
@@ -70,7 +70,7 @@ func NewDNSProviderCredentials(nameserver, tsigAlgorithm, tsigKeyName, tsigSecre
 	}
 
 	if tsigAlgorithm == "" {
-		tsigAlgorithm = dns.HmacMD5
+		tsigAlgorithm = dns.HmacSHA512
```

Comment:
This is a breaking change and violates our API compatibility guarantees - we can't just change default values for fields like this :)